docs/schema.md

```diff
@@ -160,6 +160,7 @@ The defined database prefixes and their "home" databases are:
 | `GSD` | The GSD database. Serving the shared format [here](https://github.com/cloudsecurityalliance/gsd-database). |
 | `GHSA` | The GitHub Security Advisory database. Serving the shared format [here](https://github.com/github/advisory-database). |
 | `LBSEC` | The LoopBack Advisory Database. Serving the shared format [here](https://github.com/loopbackio/security/tree/main/advisories). |
+| `PHSA` | The [VMWare Photon Security Advisory Database](https://github.com/vmware/photon/wiki/Security-Advisories). Serving the shared format [here](https://github.com/captn3m0/photon-os-advisories). |
 | `DSA`/`DLA`/`DTSA` | The [Debian Security Advisory](https://www.debian.org/security/) database. Serving the shared format [here](https://storage.googleapis.com/debian-osv/dsa-osv/). |
```

Comment:
We aren't keeping any particular order here but I think we should append new ones to the end in that case.

```suggestion
| `DSA`/`DLA`/`DTSA` | The [Debian Security Advisory](https://www.debian.org/security/) database. Serving the shared format [here](https://storage.googleapis.com/debian-osv/dsa-osv/). |
| `PHSA` | The [VMWare Photon Security Advisory Database](https://github.com/vmware/photon/wiki/Security-Advisories). Serving the shared format [here](https://github.com/captn3m0/photon-os-advisories). |
```

Reply:
Was trying to avoid conflicts with #104. Applied suggestion 👍🏽